diff --git a/CMakeLists.txt b/CMakeLists.txt
index 9ba7b8b..73a6653 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -233,7 +233,7 @@ endif()
 
 if(DESKTOP)
   # Use the static versions of the OpenSSL libraries.
-  set(OPENSSL_USE_STATIC_LIBS TRUE)
+  # set(OPENSSL_USE_STATIC_LIBS TRUE)
   if (MSVC)
     # Get the correct version of the OpenSSL libraries based on building for MT.
     if ("${CMAKE_CXX_FLAGS_DEBUG}" MATCHES "/MTd" OR
@@ -372,7 +372,9 @@ endif()
 # Some of the external libraries are not used for mobile.
 if(DESKTOP)
   # Build curl as a static library
-  set(CURL_STATICLIB ON CACHE BOOL "")
+  # set(CURL_STATICLIB ON CACHE BOOL "")
+  # Curl looks at BUILD_SHARED_LIBS now.
+  set(BUILD_SHARED_LIBS OFF CACHE BOOL "" FORCE)
   if (WIN32)
     # Enable Windows native SSL/TLS in libcurl.
     set(CMAKE_USE_SCHANNEL ON CACHE BOOL "")
@@ -391,6 +393,7 @@ if(DESKTOP)
     set(CMAKE_TOOLCHAIN_FILE /dev/null)
   endif()
   add_external_library(curl)
+  # TODO: Should "BUILD_SHARED_LIBS" be reset to its previous value here to avoid conflicts?
 
   add_external_library(libuv)
 
