diff --git a/CMakeLists.txt b/CMakeLists.txt
index ccd4ea8..e7a0ef8 100644
--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -20,7 +20,8 @@ option(JWT_EXTERNAL_PICOJSON "Use find_package() to locate picojson, provided to
 option(JWT_DISABLE_BASE64 "Do not include the base64 implementation from this library" OFF)
 option(JWT_DISABLE_PICOJSON "Do not provide the picojson template specialiaze" OFF)
 
-set(JWT_SSL_LIBRARY_OPTIONS OpenSSL LibreSSL wolfSSL)
+# "manual" added by Danny. Will cause no search for libraries or weirdness in the export part
+set(JWT_SSL_LIBRARY_OPTIONS OpenSSL LibreSSL wolfSSL manual)
 set(JWT_SSL_LIBRARY OpenSSL CACHE STRING "Determines which SSL library to build with")
 set_property(CACHE JWT_SSL_LIBRARY PROPERTY STRINGS ${JWT_SSL_LIBRARY_OPTIONS})
 
